src/scrollMenu.tsx

```diff
@@ -388,8 +389,14 @@ export class ScrollMenu extends React.Component<MenuProps, MenuState> {
     } = this.checkFirstLastItemVisibility({ translate: translateProp });
     newState.firstItemVisible = firstItemVisible;
     newState.lastItemVisible = lastItemVisible;
-    newState.leftArrowVisible = !firstItemVisible;
-    newState.rightArrowVisible = !lastItemVisible;
+
+    if (hideSingleArrow) {
+      newState.leftArrowVisible = !firstItemVisible;
+      newState.rightArrowVisible = !lastItemVisible;
+    } else {
+      newState.leftArrowVisible = firstItemVisible;
```

Comment:
In this case if `hideSingleArrow=false` and first item not visible left arrow will be hidden. So can't scroll left by arrow.

Probably something like `newState.leftArrowVisible = hideSingleArrow ? !firstItemVisible : true`
Same apply for right arrow.